provisioning/cli-dist/release_role.tf

```diff
@@ -0,0 +1,65 @@
+# IAM role which allows production release workflow push artifacts to cli-dist S3 bucket
+
+data "aws_iam_policy_document" "cli_dist_release_assume_policy_doc" {
+  version = "2012-10-17"
+  statement {
+    sid = "AllowAssumeRoleFromGithub"
+    actions = [
+      "sts:AssumeRoleWithWebIdentity"
+    ]
+    effect = "Allow"
+    principals {
+      identifiers = [
+        var.github_oidc_provider_arn
+      ]
+      type = "Federated"
+    }
+    condition {
+      test     = "StringLike"
+      variable = "token.actions.githubusercontent.com:sub"
+      values = [
+        "repo:keboola/keboola-as-code:ref:refs/tags/*",
+      ]
+    }
+    condition {
+      test     = "StringEquals"
+      variable = "token.actions.githubusercontent.com:aud"
+      values = [
+        "sts.amazonaws.com"
+      ]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "cli_dist_release_resources_policy_doc" {
+  version = "2012-10-17"
+  statement {
+    effect = "Allow"
+    actions = [
```

Comment:
Co jsem to tak procházel tak release workflow by mělo pouze uploadovat do cli dist S3 bucketu. Nejsem si jistý ale jestli třeba i nečte z něj v případě použití nějakého syncu. Dávám tedy radši práva na celý bucket.

Reply:
Zaroven to aj cita, pripaja sa S3 cez FUSE ako adresa a robi sa re-indexacia balickov pre Linux:
https://github.com/keboola/keboola-as-code/blob/281273c48a6fa96c187e3d180eef7f83512e7383/.github/workflows/push.yml#L434-L437

| Dávám tedy radši práva na celý bucket.
:heavy_check_mark: